src/AzureDevOpsDemoBuilder/Services/ProjectService.cs

```diff
@@ -1694,32 +1694,28 @@ private void CreatePullRequest(Project model, string pullRequestJsonPath, AzureD
                     pullRequestJsonPath = model.ReadJsonFile(pullRequestJsonPath);
                     pullRequestJsonPath = pullRequestJsonPath.Replace("$reviewer$", model.Environment.UserUniqueId);
                     Repository objRepository = new Repository(_workItemConfig);
-                    string[] pullReqResponse = new string[2];
 
-                    pullReqResponse = objRepository.CreatePullRequest(pullRequestJsonPath, repositoryId);
-                    if (pullReqResponse.Length > 0)
+                    (string pullRequestId, string title) = objRepository.CreatePullRequest(pullRequestJsonPath, repositoryId);
```

Comment:
I'm not familiar with this syntax... But if it compiles, looks good to me :)

Reply:
yes, I works Dave.